src/app/store.ts

```diff
@@ -1,6 +1,15 @@
 import { combineSlices, configureStore } from '@reduxjs/toolkit';
+import todosReducer from '../features/todos';
+import currentTodoReducer from '../features/currentTodo';
+import userReduser from '../features/users';
```

Comment:
There is a typo in the import statement: `userReduser` should be `userReducer`. This needs to be corrected to ensure the reducer is correctly imported and used in the `rootReducer`.